source/css/_common/components/sidebar/site-state.styl

```diff
@@ -7,7 +9,6 @@
 }
 
 .site-state-item {
-  display: inline-block;
```

Comment:
[![1.gif](https://i.loli.net/2019/03/17/5c8e2eae70937.gif)](https://i.loli.net/2019/03/17/5c8e2eae70937.gif)

Reply:
@1v9 It's not about `display: inline-block`. I forgot to add `!important` to `display: flex` when `motion` enabled. I'll fix it

Reply:
https://github.com/julianshapiro/velocity/issues/672